sdk/go/x/auto/errors_test.go

```diff
@@ -27,6 +27,7 @@ import (
 )
 
 func TestConcurrentUpdateError(t *testing.T) {
+	t.Skip("disabled")
```

Comment:
Why was this disabled?

If this is really necessary can we open an issue and link it here?